torch/fx/passes/split_module.py

```diff
@@ -31,6 +31,67 @@ def split_module(
     root_m: torch.nn.Module,
     split_callback: Callable[[torch.fx.node.Node], int],
 ):
+    """
+    Creates subgraphs out of main graph
+
+    Args:
+        m (GraphModule): Graph module to split
+        root_m (torch.nn.Module): root nn module. Not currently used. Included
+            because the the root nn module is usually transformed via
+            torch.fx._symbolic_trace.symbolic_trace, see example below
```

Comment:
If this isn't used we should probably just delete it, but we can do it in a follow-up